.github/workflows/ci.yml

```diff
@@ -3,15 +3,9 @@ name: CI
 on:
   push:
     paths-ignore:
-      - '**/*.md'
-      - '**/*.txt'
-    branches:
-      - main
-      - release/**
-  pull_request:
-    paths-ignore:
-      - '**/*.md'
-      - '**/*.txt'
+      - '**.md'
+      - '**.txt'
```

Comment:
I've realized '**' is implemented as that it just matches everything including '/' (while '*' only matches any character except '/') - thus no need for '/*' in the pattern